br.com.wiselabs.simplexity.appdata.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 Franklin Ronald -->
+<component type="desktop-application">
+    <id>br.com.wiselabs.simplexity</id>
+    <metadata_license>0BSD</metadata_license>
+    <project_license>BSD-3-Clause</project_license>
+    <name>Simplexity</name>
+    <summary>Simple desktop app for Perplexity.AI</summary>
+    <developer id="br.com.wiselabs">
+        <name>Wiselabs Software</name>
+    </developer>
+    <description>
+        <p>The non-nonsense simple desktop app for Perplexity AI. It's just work, no magic, no tray, no scripts injected and nothing obfuscated.</p>
+    </description>
+    <launchable type="desktop-id">br.com.wiselabs.simplexity.desktop</launchable>
+    <categories>
+        <category>Office</category>
+        <category>Network</category>
+    </categories>
+    <keywords>
+        <keyword>Perplexity.AI</keyword>
+        <keyword>search</keyword>
+        <keyword>chatbot</keyword>
+        <keyword>ai</keyword>
+    </keywords>
+    <screenshots>
+        <screenshot type="default">
+            <caption>Screenshot main window</caption>
+            <image>
+                https://raw.githubusercontent.com/Wiselabs/simplexity/master/img/screenshot.png
```

Comment:
this should never point at a tip of a branch. It should be a specific revision or tab.

Reply:
Done.